tests/mli/test_worker_manager.py

```diff
@@ -184,7 +185,12 @@ def test_worker_manager(prepare_environment: pathlib.Path) -> None:
     # create a mock client application to populate the request queue
     msg_pump = mp.Process(
         target=mock_messages,
-        args=(work_queue, file_system_store, fs_path, comm_path),
+        args=(
+            config_loader.get_queue(),
```

Comment:
Does this work? It seems like we would need to put a valid value in env vars for the queue and feature store, but never do so.

This test isn't a test of config loader, so consider monkeypatching:

```py
with monkeypach.context() as ctx:
  ctx.setattr(config_loader, "get_queue", work_queue)
  ctx.setattr(config_loader, "get_feature_store", file_system_store)
```

Reply:
I think it likely works because if the environment variable aren't set, they just return `None`. But I'll fix the test!